controller/src/beerocks/cli/beerocks_cli_bml.h

```diff
@@ -80,7 +80,7 @@ class cli_bml : public cli {
 
 private:
     // Help functions
-    void setFunctionsMapAndArray();
+    void setFunctionsMapAndArray() override;
```

Comment:
Again, shouldn't we use `virtual`?
```suggestion
    virtual void setFunctionsMapAndArray() override;
```